packages/react-vapor/src/components/table-hoc/TableHeaderWithSort.tsx

```diff
@@ -64,7 +64,7 @@ export class TableHeaderWithSort extends React.Component<
         });
 
         return (
-            <th className={headerCellClasses} onClick={() => this.props.onSort()}>
+            <th id={this.props.id} className={headerCellClasses} onClick={() => this.props.onSort()}>

```

Comment:
I think I would also add the `tableId`, just in case multiple tables in the same page use the same `id` for their rows.

Reply:
I could add the attribute tableId and Id but seperate them, rather than mixing them:

`tableId={this.props.tableId}` id={this.props.id}` 
Since i'm using `this.props.id` to the attribute id, i prefer not to mix them in the same attribute